### DISCUSSION TITLE
How to select with an input having duplicate values?

### DISCUSSION THREAD

**Discussion by Parkersungwoo** [cypress-io/cypress#17974] [Questions and Help]
Right now I have an input with the dropdown menu that looks similar to this:
![Screen Shot 2021-09-02 at 4 54 18 PM](https://user-images.githubusercontent.com/31974842/131914743-af77cbb4-5c7a-449b-b8ee-c018e4e53115.png)

How can I select the first instance of the value `18z7gde6 - Digital China (China) Ltd. - Distributor`?

This is the error when I run the `cy.select()`.
![Screen Shot 2021-09-02 at 4 54 06 PM](https://user-images.githubusercontent.com/31974842/131914718-2859f8ec-3d3a-40d7-a0df-344d572fc764.png)

 I've tried using `.first()` but since the `select()` fails the `first()` call is never triggered. Thanks!

**Comment by bahmutov** [comment]
What’s the html markup for that widget? These options all have the same value just different text?

**Comment by Parkersungwoo** [comment]
Here is the HTML: ![Screen Shot 2021-09-03 at 11 33 53 AM](https://user-images.githubusercontent.com/31974842/132031076-60db2a9c-4bb3-4325-98b0-74fff18955bb.png)

Note - this `select` will have a varying number of options across tests so I cannot use `value` in this instance to select.

**Reply by bahmutov** [reply]
But why the same two labels (the ones ending with the `- Distributor`) have different values 6 and 8? How does the user know what to select?
